packages/utils/src/getAccountsFromLocalStorage.ts

```diff
@@ -0,0 +1,22 @@
+import { AuthorizeResponse } from '@deriv/api-types';
+
+type TLocalStorageAccountsList = {
+    [k: string]: {
+        token: string;
+        accepted_bch: number;
+        landing_company_shortcode: string;
+        residence: string;
+        session_start: number;
+    } & NonNullable<NonNullable<NonNullable<AuthorizeResponse['authorize']>['account_list']>>[number];
+};
+
+/**
+ * Gets the current user `accounts` list from the `localStorage`.
+ */
+const getAccountsFromLocalStorage = () => {
+    const accounts: TLocalStorageAccountsList = JSON.parse(localStorage.getItem('client.accounts') || '{}');
+
+    return accounts;
```

Comment:
```suggestion
    const data = localStorage.getItem('client.accounts');

    // If there is no accounts list, return undefined.
    if (!data) return;

    const accounts: TLocalStorageAccountsList = JSON.parse(data);
```

Reply:
Probably needs to `return {}` if there is no `account_list` (I'm basing this off the test)

Reply:
Better to return `undefined` so TS will enforce a check while using it, We can improve the test 🤔